src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEventSummary.kt

```diff
@@ -82,6 +85,22 @@ object HoppityEventSummary {
     private val storage get() = ProfileStorageData.profileSpecific
     private val liveDisplayConfig get() = config.eventSummary.liveDisplay
     private val updateCfConfig get() = config.eventSummary.cfReminder
+    private val disallowListHoppityIslands by lazy {
```

Comment:
cant we use a list of allowd islands instead, and load that from the repo via the egg location data? or is this check different here?